sds/src/match_validation/config.rs

```diff
@@ -100,7 +100,7 @@ pub struct HttpValidatorOption {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
 pub struct CustomHttpConfig {
     pub endpoint: String,
-    #[serde(default)]
+    #[serde(default = "default_hosts")]
```

Comment:
You can just use `#[serde(default)]` if you want since `Vec` [implements](https://doc.rust-lang.org/src/alloc/vec/mod.rs.html#3811) `Default` (which is an empty Vec)